web/satellite/src/components/account/billing/paymentMethods/TokenTransactionItem.vue

```diff
@@ -1,13 +1,146 @@
-// Copyright (C) 2019 Storj Labs, Inc.
+// Copyright (C) 2022 Storj Labs, Inc.
 // See LICENSE for copying information.
 
 <template>
-    <div class="container" />
+    <div class="container">
+        <div class="divider" />
+        <div class="container__row">
+            <div class="container__row__item__date-container">
+                <p class="container__row__item date">{{ billingItem.start.toLocaleDateString() }}</p>
+                <p class="container__row__item time">{{ billingItem.start.toLocaleTimeString([], {hour: '2-digit', minute: '2-digit'}) }}</p>
+            </div>
+            <div class="container__row__item__description"> 
+                <p class="container__row__item__description__text">CoinPayments {{ billingItem.description.includes("Deposit")? "Deposit": "Withdrawal" }}</p>
+                <p class="container__row__item__description__id">{{ billingItem.id }}</p>
+            </div>
+            <p class="container__row__item amount">
+                <b>
+                    <span v-if="billingItem.type === 1">
+                        ${{ billingItem.quantity.received.toFixed(2) }}
+                    </span>
+                    <span v-else>
+                        ${{ billingItem.quantity.total.toFixed(2) }}
+                    </span>     
+                </b>
+            </p>
+            <p class="container__row__item status">
+                <span :class="`container__row__item__circle-icon ${billingItem.status}`">
+                    &#9679;
+                </span>
+                {{ billingItem.formattedStatus }}
+            </p>
+            <p class="container__row__item download">
+                <a v-if="billingItem.link" class="download-link" target="_blank" :href="billingItem.link">View On CoinPayments</a>
+            </p>
+        </div>
+    </div>
 </template>
 
 <script lang="ts">
-import { Vue } from 'vue-property-decorator';
+import { Prop, Vue } from 'vue-property-decorator';
 
+import { PaymentsHistoryItem } from '@/types/payments';
+
+// @vue/component
```

Comment:
I thought that we are leaving just @Component. Not sure if it is working like this. Please test it. If so, LGTM

Reply:
Hi Nikolai, I somehow missed this comment, but adding that `@Component` resolved a long standing issue for me! Thanks!